shaky/src/main/java/com/linkedin/android/shaky/ShakeDelegate.java

```diff
@@ -69,4 +69,11 @@ public void collectData(Activity activity, Result data) {
      * This method can be overridden to send data to a custom URL endpoint, etc.
      */
     public abstract void submit(Activity activity, Result result);
+
+    /**
+     * @return true if more attachments are allowed to be attached to the reports
+     */
+    public boolean isAddAttachmentFeatureEnabled() {
```

Comment:
lets call this something like "shouldAllowAdditionalAttachments" or something, to make it a bit more expressive of what this method indicates

Reply:
OK